src/App.js

```diff
@@ -0,0 +1,27 @@
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
+
+function App() {
+  const [greeting, setGreeting] = useState('');
+
+  useEffect(() => {
+    axios.get('http://localhost:3001/api/v1/random_greeting')
+      .then((response) => {
+        setGreeting(response.data.message);
+      })
+      .catch((error) => {
+        throw new Error('Error fetching random greeting:', error);
+      });
+  }, []);
+
+  return (
+    <div className="App">
+      <header className="App-header">
+        <h1>This App Shows You a greeting!!!</h1>
+        <p>{greeting}</p>
+      </header>
+    </div>
+  );
+}
```

Comment:
As mentioned in the [requirements](https://github.com/microverseinc/curriculum-rails/blob/main/connect-frontend-frameworks/hello_world_two_apps.md#:~:text=Create%20the%20Greeting%20component%20that%20will%20display%20a%20greeting.%20Set%20it%20up%20as%20a%20route%20in%20your%20App%20component.), you should have a route that leads to the greetings component in the app `component`.